boards/shields/st_b_lcd40_dsi1_mb1166/st_b_lcd40_dsi1_mb1166.overlay

```diff
@@ -18,3 +33,36 @@
 		rotation = <90>;
 	};
 };
+
+&zephyr_ltdc {
+	status = "okay";
+	width = <800>;
+	height = <480>;
+	def-back-color-red = <0>;
```

Comment:
These properties are specific to the STM32 LTDC node, so they should be placed on board specific overlays. The idea is that if someone chose to design a board that could connect to this display module and used an NXP chip to drive the display instead of an ST one, the properties defined in the `zephyr_ltdc` and `zephyr_mipi_dsi` node would be shared between both chips. The SOC-specific settings should go into the board overlays for the shield.

The properties I can see that need to be moved to a board overlay are the following:
- `def-back-color*`
- `loosely-packed`
- `largest-packet-size`
- `bta-ack-disable`

Reply:
Yes, but a lot of the properties on the LTDC are shared between other LCD controllers (such as NXP's LCDIF). Most of the properties there are ok to stay- in theory this display could likely be enabled with an NXP SOC (or any SOC with a DSI controller supporting video mode)

I completely understand the odds are *low* that someone is going to buy an NXP SOC and say to themselves "you know what, I think I'll purchase a display module sold by ST, and validate my board with that". However, because we are using these generic `zephyr_lcdif` node labels, we are implicitly committing to only placing properties shared between *all* parallel LCD controllers (and MIPI DBI) controllers on those nodes.

I did some work to standardize a lot of the LCD properties here: https://github.com/zephyrproject-rtos/zephyr/pull/53907. I think the same could be done for the MIPI DSI controllers, looking at the controller on the RT500 I believe I see a use for the following properties:
- `bta-ack-disable`
- `largest-packet-size`
- `loosely-packed`

If we want to try to standardize these properties I'd welcome that. At the end of the day, all these controllers need to do the same thing, so a lot of properties can be shared. However, I don't want to bog this PR down with that- for now, probably we should just move the properties to board overlays?